content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1014 @@
+---
+slug: "stats19"
+title: "stats19: a package for road safety research"
+package_version: 0.2.0
+authors:
+  - Layik Hama
+  - Robin Lovelace
+date: 2019-02-18
+categories: blog
+topicid:
+tags:
+- R
+- software peer review
+- data access
+- geospatial
+- transport
+- packages
+- stats19
+- road safety 
+- open data UK
+---
+
+#### Introduction
+
+`stats19` is a new R package enabling access to and working with
+Great Britain’s official road traffic casualty database,
+[STATS19](https://data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data).
+
+We started the package in late 2018 following three main motivations:
+
+1.  The release of the 2017 road crash statistics, which showed
+    worsening road safety in some areas, increasing the importance of
+    making the data more accessible.
+2.  The realisation that many researchers were writing add-hoc code to
+    clean the data, with a huge amount of duplicated (wasted) effort and
+    potential for mistakes to lead to mistakes in the labelling of the
+    data (more on that below).
+3.  An understanding of the concept of ‘modularity’ in software design,
+    following the [Unix
+    philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) that
+    programs should ‘do one thing and do it well’. This realisation has
+    led to code inside the rOpenSci-hosted package
+    [`stplanr`](https://github.com/ropensci/stplanr) being split-out
+    into 2 separate packages already:
+    [`cyclestreets`](https://github.com/Robinlovelace/cyclestreets) and
+    [`stats19`](https://github.com/ropensci/stats19).
+
+We have a wider motivation: we want the roads to be safer. By making
+data on the nature of road crashes more publicly accessible to inform policy,
+we hope this package saves lives.
+
+`stats19` been tested and peer reviewed thanks to rOpenSci and is now published in the
+Journal of Open Source Software
+([JOSS](http://joss.theoj.org/papers/10.21105/joss.01181)) (Lovelace et
+al. 2019), making this an ideal time to present the package to the world 🚀.
+For installation and the code, see its home in rOpenSci:
+<https://github.com/ropensci/stats19>, and the package documentation at
+<https://itsleeds.github.io/stats19/>.
+
+In this post, we’ll provide a bit of context, show how the package works, and provide ideads for future work building on the experience. Version [`0.2.0`](https://cran.r-project.org/web/packages/stats19/index.html) has just been release on CRAN, which contains a few improvements, some of which are used in this blog post (`ask = FALSE` in `get_stats19()`, for example, which makes it even quicker to get data with this package).
+
+#### Short history and affiliations
+
+The main authors of `stats19` are based at the [Institute for Transport Studies
```

Comment:
@Robinlovelace @layik There are a few items like this one that you haven't commented on. Can you please respond so I know if I have misunderstood, if you disagree, or if you missed a particular comment? I'm 100% open to you disagreeing with my suggestions :-)

Reply:
@stefaniebutland GitHub is just becoming too smart. I literally just saw this! Working on it.

Reply:
Done.

Reply:
Since "Malcolm" is not an author on this post, best to say "Robin Lovelace" and "Malcolm Morgan"

My big remaining concern is that Mark Padgham is the only package author not mentioned here. Is there a reason for this? It makes this paragraph seem biased still. https://github.com/ropensci/stats19/graphs/contributors

Reply:
Right, I will add the surnames, Mark is a package author/contribute but neither he nor I have done any prior work on this as far as I know. Maybe section header or some other reference to history would make this less "biased" or I can tag Mark to say he is fine :)